ci/test/add

```diff
@@ -3,7 +3,7 @@ set -ex
 
 function cleanup {
   cd /tmp
-  rm -rf test_folder/
+  sudo rm -rf test_folder/
```

Comment:
instead of running with sudo, we can delete via docker (i.e. bind-mount /tmp, then run `rm -r` inside the container), as it is done in the [`test/common.sh`](https://github.com/systemslab/popper/blob/master/ci/test/common#L6) script. BTW, we can refactor that and reuse that function